### DISCUSSION TITLE
How to use py::bool_ in nanobind?

### DISCUSSION THREAD

**Discussion by harsszegi** [wjakob/nanobind#131] [Q&A] [answered]
Hi,
I need to create something like this: https://github.com/pybind/pybind11_json
for nanobind and for rapidjson.

I fail to understand how to construct a "bool" binding object with nanobind.
How to create a bool object currently?
Also as I saw "::float_" was also dropped in favor of "::int_", am I getting it right?
Thanks,

**Comment by wjakob** [comment, marked as answer]
Please make sure that you have read through all of the latest documentation, especially the section on exchanging information between Python and C++.

`float_` was not yet ported from pybind11, you are welcome to propose such a wrapper in a PR.

**Reply by harsszegi** [reply]
Sorry, I tried go through docs/, but all I could find "tensor.md" which talks about exchanging information. I also tried looking in the removed section, but couldn't find any information about how to convert from "py::bool_" to nanobind. Could you point me to the appropriate documentation which describes this?
Thanks,

**Reply by wjakob** [reply]
You should really read the whole documentation before embarking on the project you mentioned.
The part I was talking about is here: https://nanobind.readthedocs.io/en/latest/exchanging.html

**Reply by harsszegi** [reply]
Ah sorry, I checked the "docs" folder in the repo.
